### PR TITLE
Rename root package + publish snapshots

### PR DESCRIPTION
**Pull request by ktonga**
ping @NeQuissimus I'll need those credentials of yours now :smiley:

### CODE REVIEW

**Comment by ktonga** [comment]
It shuoldn't be trying to publish, yet. Why `TRAVIS_PULL_REQUEST` is false if this IS a pull request?

**Comment by ktonga** [comment]
Oh the check that failed is `push`. Why do we have two checks (push and pr) which look to be running the same job? Should `push` be running only for merges to `master`?

**Comment by ktonga** [comment]
Can anyone with admin rights take a look at checks config to see what's going on? If we merge this PR without fixing that first we will be publishing snapshots for each commit in the PRs as well.

**Comment by jdegoes** [comment]
@ktonga Try now. You should be admin.

**Comment by NeQuissimus** [comment]
@ktonga 

```
[nix-shell:~/dev/scalaz-zio]$ travis encrypt SONATYPE_USERNAME=<USER>
Detected repository as scalaz/scalaz-zio, is this correct? |yes| yes
Please add the following to your .travis.yml file:

  secure: "1KShGbny4TgD86w5/huvJEwcLYEQKjiUZeornS8W0rt8+j2emfFpb2VoyCfOQmXQrh0t5N96dS/hT0C+cDE0+/12rOE4zpm5RDDT7skY+ayTb3SKkIP9Qk5behvyIZi8xc5ny6z6UZoFBeTGJMZQgeuADhfcQHw68cZl4kEM9fU8L6Yc8jKUN6xTENnVpIDmM29Tbx/WyNA9EKw+1CL2fJVVLmvsQphieYoVjQfhZjl1+M289cZ+dX1UjsuvfrfwBTwFoabnZbQATZ9ALvwIfuOKYIdapS3rAayaKC7uihWZmCnNT4mmleob+zXdUjOWl/GXlHAgeCWU1GP7qZ65aLRx+URs/Vh+VAi5gXceF9Mka0VYMb/nm+noJIy9WLKa4kfnHruqR5NzxTq8D0xkgSa2v1tRCT7Vf98+RlzPTVF+rG/8lGa/nFoxqAkbB8ycTYSoAIswan4lx2PhD2GPWsCdiUBrMljPTr2ie88x45w4b5f5eZbsxc1qi2ks1oFTVJhk36l9+E+V8iFi3uZbs2x5Qfar7301mWwarcwfxt/CgDqSz4naWXyPfRbh9KwwFnjl2yEiYoPb0zU8eKhBid+JBi94gvlm2ykmHTYWD39govSgW69PSBoW8JROE10skpiLFsowe/U6FkVNA+tWavk/PNHPpHb4dlP63mbiOTU="

Pro Tip: You can add it automatically by running with --add.

[nix-shell:~/dev/scalaz-zio]$ travis encrypt SONATYPE_PASSWORD=<PW>
Please add the following to your .travis.yml file:

  secure: "RC4sr+jES1TmvcYaTLlPEFjf5HRjNqO5oKS8NtmqNfjcnyiQu/xDfGYQ3cy7hJ3oX4PlzZMaCpA9FEq4/PNT5CwoZcwUPUsNLAsGpn4he/y/4gWURrsI/0G0dtTRcu6NHvl7kL2o+SWQZD1qMOgwcdwj2bw5fqbPtRKAVYadS/oYvb9XnZsIgC7wGzpAY0L/RFh/KifCl6n1gRl28/6TP1lx8Au6UpKjMfKVQcNcoZKERQCyrVhjeqWFwQiZP+7wHFyFoi717wJLMN7eRLxts2xC0lyYOfQlWSrH4jB1XikPfJtGgj6nR3FBd6W0RS2ekmG/ILNnpiIrO9OX8cVO2SZ9+xgbdNZCxTc9LHTDpD015n9qgSnpcuZxHMgkU+/R3v80ss/h0EWG/qmh53xNBc5E+rjIKwcVMjlMrNyGad8vpabz3XdOzk67hcRit/pDxEYzcw5VMOv+kxzZWy3u2+obm6bk+JzS30Kw04qneXdHLTbcXLglRoyVKweNR5aINWItLuqhQbpJ1JOjHz+mMhaMeCBLHsyeakipIkRGTmMmuuxpip2AkgYjhC7V8uLhQIitODN6FJ1K5y7UADYHGCsp5+av/cFGVuDnfRV1PRbC7oz5yHkdOopwBFOHqMZ+TmAu5133t5aazVlcHJuoy/Rfplg+pcS+rkoFZcQjVGI="

Pro Tip: You can add it automatically by running with --add.
```

**Comment by ktonga** [comment]
Thank you @NeQuissimus I'll try them as soon as I figure out why we are triggering `push` Travis check on PRs instead of just `pr`

**Comment by ktonga** [comment]
@NeQuissimus I've updated the travis file with those values but it fails

```
[error] Unable to find credentials for [Sonatype Nexus Repository Manager @ oss.sonatype.org].
```

Anything else that needs to be done?

**Comment by NeQuissimus** [comment]
Check https://github.com/scalaz/scalaz/blob/series/8.0.x/build.sbt and https://github.com/scalaz/scalaz/blob/series/8.0.x/project/ScalazBuild.scala

You need to add the Sonatype credentials to the build.

**Comment by ktonga** [comment]
I did it as part of this PR https://github.com/scalaz/scalaz-zio/pull/20/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR17

File: build.sbt [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+}
+
+dynverSonatypeSnapshots in ThisBuild := true
+
+lazy val sonataCredentials = for {
+  username <- sys.env.get("SONATYPE_USERNAME")
+  password <- sys.env.get("SONATYPE_PASSWORD")
+} yield Credentials("Sonatype Nexus Repository Manager", "oss.sonatype.org", username, password)
+
+credentials ++= sonataCredentials.toSeq
```
Comment (NeQuissimus) [inline build.sbt]: `credentials in ThisBuild`, otherwise only your root project will have them

**Comment by NeQuissimus** [comment]
I left a comment

File: build.sbt (hunk 1 above)
Comment (ktonga) [reply build.sbt]: Thanks a lot man, that did the trick! What stupid implementation detail leak the `in ThisBuild` think.

**Comment by ktonga** [comment]
Hey @jdegoes good news, [habemus snapshot ](https://oss.sonatype.org/#nexus-search;quick~scalaz-zio)! You good to merge?

**Comment by jdegoes** [comment]
🕺🕺🕺👏👏👏❤️❤️❤️

**Pull request merged**